### PR TITLE
 Fix some weird behaviors when parsing empty config files

### PR DESCRIPTION
**Pull request by mcg-web**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documented?   | no
| Fixed tickets | none
| License       | MIT

### CODE REVIEW

File: Tests/Functional/Type/DefinitionTest.php [hunk 1]
```diff
@@ -53,12 +53,7 @@ public function testDefinesAnObjectTypeWithDeprecatedField()
 
     private function isDeprecated($node)
```
Comment (renatomefi) [inline Tests/Functional/Type/DefinitionTest.php]: Maybe the whole function can be removed?
it looks easier to call `$field->isDeprecated()`

Comment (mcg-web) [reply Tests/Functional/Type/DefinitionTest.php]: Nice catch :+1:  thank you

Comment (renatomefi) [reply Tests/Functional/Type/DefinitionTest.php]: Cool, are you going to remove it? I'm looking forward to merge it! :)

Comment (mcg-web) [reply Tests/Functional/Type/DefinitionTest.php]: yes i'm going to remove it

**Comment by renatomefi** [comment]
Looks great :+1:

**Pull request merged**